### PR TITLE
Perimeter Calculator

### PR DESCRIPTION
**Pull request by poorvadiwan**
I went through the Maths section of the repository and found programs for volume and area but not perimeter. So this is a program to calculate perimeter of geometrical objects.

- [x] Add an algorithm?

- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Java/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized.
- [x] All functions and variable names follow Java naming conventions.

### CODE REVIEW

File: src/main/java/com/thealgorithms/maths/Perimeter.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    // Perimeter of different 2D geometrical shapes
+
+    /**
+     *Calculate the Perimeter of polygon.
+
+     * @parameter length of side.
+     * @parameter number of sides.
+     * @return Perimeter of given polygon
+     */
+    public static float perimeter_polygon( int n, float side){
```
Comment (siriak) [inline src/main/java/com/thealgorithms/maths/Perimeter.java:13]: Please fix formatting

File: src/main/java/com/thealgorithms/maths/Perimeter.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        return perimeter;
+    }
+
+    /**
+     *Calculate the Perimeter of rectangle.
+
+     * @parameter length and breadth.
+     * @return Perimeter of given rectangle
+     */
+    public static float perimeter_rectangle( float len, float brea){
```
Comment (siriak) [inline src/main/java/com/thealgorithms/maths/Perimeter.java:24]: Please use full words instead

File: src/main/java/com/thealgorithms/maths/Perimeter.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+     *Calculate the circumference of circle.
+
+     * @parameter radius of circle.
+     * @return circumference of given circle.
+     */
+    public static double circumference( float r){
+        double circumference = 2*Math.PI*r;
+        return circumference;
+    }
+
```
Comment (siriak) [inline src/main/java/com/thealgorithms/maths/Perimeter.java:39]: No need for this newline

**Review by siriak — CHANGES_REQUESTED** [review]

**Review by siriak — COMMENTED** [review]
Please also add some JUnit tests

**Comment by poorvadiwan** [comment]
I have completed the changes you asked!! 
I apologize for the mess, actually, I am new to GitHub so I was a bit confused.

**Review by siriak — APPROVED** [review]
Looks good, thanks!

**Pull request merged**